manual/program/reviewer_guidelines.rst

```diff
@@ -0,0 +1,167 @@
+
+.. image:: reading.png
+  :width: 400
+  :alt: I don't always do research. But when I do I make sure it's qualitative
+
+You should analyze the presentation in the aspects described below, thanks to `Doug Hellmann <https://doughellmann.com/blog/>`_ for these tips:
+
+The Abstract: occasionally a title comes along that is so compelling that I have to remind myself to keep reading before voting +1 and moving on to the next talk. It isn't enough for a proposal to cover an interesting topic. It has to indicate that the talk will be interesting, too. While I am reading, I look for several factors.
+
+- **Is the abstract clear?** The speaker should describe the topic they plan to talk about in terms I can understand, even if I don't know anything about that subject area. A clearly written abstract, without a lot of domain-specific jargon, tells me the speaker will be able to communicate with the audience.
+
+- **Is the abstract complete?** An incomplete proposal is the largest negative factor I consider. If a proposal is incomplete, I can't really say what the speaker will talk about, or even if they know the subject matter for their talk. If a proposal does not have a detailed summary or outline, I as the submitter to provide more detail.
+
+- **Is the abstract compelling?** Without regard to the actual subject, is the abstract written in a way to attract an audience? I look for an abstract that shows the speaker is excited about the topic, and that they will be conveying that same excitement to the audience.
+
+The Topic: for some people, the subject matter of a talk is the most important, or only, aspect taken into consideration when voting. I have seen presentations on topics I thought would be boring, but which were delivered with such enthusiasm that I enjoyed them more than talks I thought would be interesting from the outset. In my mind, the topic is an important factor, but not necessarily the most important, to be considered.
+
+- **How relevant, immediately useful and novel is the topic?** I look first at whether the topic is relevant to the conference attendees. Attendees will have a range of experience levels, interests, and backgrounds. Although we want a broad set of topics, we do need to be careful to avoid talks that are too narrowly focused on a niche. I often recommend that new projects which show a lot of promise convert their talk proposal into a poster proposal. As a counterpoint to considering whether a topic is too niche, I also try to take into account whether the audience will take away something immediately useful.
+
+Conflict of interest
+--------------------
+
+.. image:: coi.jpeg
+  :width: 400
+  :alt: Conflict of interest? I never saw a conflict that didn't interest me
+
+The definition of a Conflict of Interest in peer review is a circumstance that makes you "unable to make an impartial scientific judgment or evaluation.” ([PNAS Conflict of Interest Policy](http://www.pnas.org/site/authors/coi.xhtml)). 
```

Comment:
```suggestion
The definition of a Conflict of Interest in peer review is a circumstance that makes you "unable to make an impartial scientific judgment or evaluation.” (`PNAS Conflict of Interest Policy <https://www.pnas.org/site/authors/coi.xhtml>`_). 
```

Okay and now as I'm out of that meeting here's the suggested change for the `.md` -> `.rst` hyperlink format.